\section{Introduction}

Understanding how environmental changes affect diversification is an important
goal in evolutionary biology, biogeography, and global change biology.
Environmental processes that operate at or above the level of communities can
simultaneously cause speciation or extinction across multiple evolutionary
lineages, and thus have a pronounced effect on the diversity and distribution
of species.
% across an area.
Island archipelagos that harbor diverse communities of co-distributed lineages
and have a relatively well-understood geological history present powerful
systems for understanding such shared processes of diversification
\citep{Gillespie2007,Losos2009,Vences2009,Rafe2013AREES}.
The Philippine Islands represent such a model system, with more than 7100
islands that arguably harbor the highest concentration of terrestrial
biodiversity on Earth
\citep{CatibogSinha2006,RafeDiesmos2009, Heaney1998,Rafe2013AREES};
how this diversity accumulated has been a topic of discussion since the
early development of the field of biogeography
\citep{Wallace1869,Huxley1868}.

The landscape of the Philippines has experienced a complex history.
Climatological oscillations, primarily during the Pleistocene, led to the
repeated formation and fragmentation of Pleistocene Aggregate Island Complexes
\citep[PAICs;][]{Inger1954,Heaney1985,RafeDiesmos2001,RafeDiesmos2009,Esselstyn2009MPE,Lomolino2010,Siler2010,Rafe2013}.
During lower sea levels of glacial periods, islands coalesced into seven major
landmasses (PAICs) that were fragmented into individual islands during
interglacial periods.
These climate-driven cycles have occurred at least six times during the last
500,000 years \citep{Rohling1998,Siddall2003}, with additional cycles
occurring in the late Pliocene and early Pleistocene
\citep{Haq1987,Miller2005}.

For nearly three decades, the repeated formation and fragmentation of PAICs has
been a prominent model of diversification in the Philippines
\citep{Inger1954,Heaney1985,RafeGuttman2002,Evans2003,Heaney2005,Roberts2006,Linkem2010,Siler2010,Siler2011,Siler2012,Rafe2013}.
However, there is growing recognition of the complexity of historical processes
that were involved in the diversification of this megadiverse archipelago.
For example,
there is evidence that older tectonic processes contributed to vertebrate
diversification on precursor paleoislands that predates the modern distribution
of landmasses in the Philippines
\citep[$\sim$30--5 mya;][]{Jansa2006,Blackburn2010,Siler2012,Rafe2013}.
Additionally, the region's biodiversity has likely been shaped further by
dispersal events from mainland source populations via recognized colonization
routes \citep[see][for a review]{Rafe2013} and finer-scale isolating
mechanisms that led to in situ diversification
\citep{Heaney2011,Linkem2011}.
Nonetheless, the question remains: Was climate-driven fragmentation of the
islands an important process of speciation? 

A model of speciation via repeated vicariance predicts that divergences across
taxa that occur on historically connected islands (i.e., islands within the
same PAIC) should tend to be clustered around times when rising sea levels
fragmented PAICs into the islands of today.
Therefore, if we compare the divergence times of multiple pairs of populations
or closely related species that occur on two islands that were connected during
glacial periods of lower sea levels, we expect some to be contemporaneous
with interglacial fragmentation events.
Such patterns of shared divergences would be difficult to explain by
other mechanisms, such as over-water dispersal.

\citet{Oaks2012} initially tested this prediction by inferring how many unique
divergence times best explained mitochondrial sequence data from 22 pairs of
populations from across the Philippines, using a model choice method based on
approximate-likelihood Bayesian computation (ABC).
However, using simulations, they found that this popular ABC approach was
demonstrably sensitive to prior assumptions, with a bias toward over-clustering
divergence times, both of which rendered the results difficult to interpret,
and potentially skewed toward interpretations of simultaneous diversification.
\citet{Oaks2014dpp} reanalyzed these data with an ABC method that alleviated
these issues; however, they found that reducing the genetic data to a small
number of summary statistics left ABC methods with little information to update
prior assumptions.

Here we use comparative genomic data and a new full-likelihood Bayesian method
to test the hypothesis that repeated fragmentation of islands was a causal
mechanism of vicariant diversification among terrestrial vertebrates in the
Philippines.
By using all the information in thousands of loci from 16 inter-island
pairs of gecko populations,
we demonstrate a new method that provides the first robust evaluation of
this central tenet of the PAIC model of diversification.
Our results support independent diversification among pairs of gecko
populations,
provide evidence against fundamental predictions of the PAIC model of
diversification,
and underscore the importance for caution against adhering to overly simplistic
models of diversification based on little evidence when studying dynamic and
biodiverse regions such as the Philippines.
% The results show that by using all of the information in genomic
% data, we are finally able to robustly evaluate this hypothesis.


\section{Methods}

\subsection{Sampling}
For two genera of geckos, \spp{Cyrtodactylus} and \spp{Gekko}, we sampled
individuals from pairs of populations that occur on two different islands.
Because the climate-mediated fragmentation of the islands was a relatively
recent phenomenon, we selected pairs of populations that were inferred to be
closely related from previous genetic data
\citep{Siler2012, Siler2014kikuchii, Welton2010, Welton2010zootaxa, Siler2010}.
In other words, we avoided pairs that we knew \emph{a priori} diverged well
before the connectivity cycles, because these cannot provide insight into
whether divergences were clustered \emph{during} these cycles.

We also sought to sample pairs that spanned islands connected during glacial
periods, as well as islands that were never connected.
We included the latter as ``controls.''
Because these islands were never connected, the distribution of the populations
inhabiting them can only be explained by inter-island dispersal.
The divergence between these populations was either due to that
dispersal, or an earlier intra-island divergence.
Either way, the timing of divergences across islands that were never connected
are not expected to be clustered across pairs.
These controls are important given the tendency for previous approaches to this
inference problem to over-estimate shared divergences
\citep{Oaks2012,Oaks2014reply}.
Finding shared shared divergence times among pairs for which there is no
tenable mechanism for shared divergences will indicate a problem and prevent us
from misinterpreting shared divergences among pairs spanning islands that were
fragmented as evidence for the PAIC model of vicariant diversification.
Applying these criteria, for both genera, we identified eight pairs of
populations, including a mix of pairs spanning islands that were never
connected, islands that were connected, and islands that were possibly
connected during glacial lowstands
(Figure~\ref{fig:map};
Table \ref{table:comparisons} \& S\ref{table:reftosampledata}).
The Islands of
Bohol and Camiguin Sur,
Palawan and Borneo,
Sibuyan and Tablas, 
and
Sabtang and Batan
are not believed to
have been fully joined during glacial periods.
However, even if these pairs of islands did not have a complete land
connection, they may have been close enough to permit some limited gene flow
given their immediate proximity and the relative dispersal ability of gekkonid
lizards.
As a result, we leave their connection status during glacial lowstands as
ambiguous 
(Figure~\ref{fig:map} \& Table~\ref{table:comparisons}).

\ifembed{
\input{fig-map.tex}
}{}

\ifembed{
\input{tables/table-comparison-summary.tex}
}{}

\subsection{Genomic library preparation and sequencing}

We extracted DNA from tissue using the guanidine thiocyanate method described
by \citet{Esselstyn2008}.
We diluted the extracted DNA for each individual to a concentration of 5
ng/$\mu$L based on the initial concentration measured with a Qubit 2.0
Fluorometer.
We generated three restriction-site associated DNA sequence (RADseq) libraries,
each with 96 individuals, using the multiplexed shotgun genotyping (MSG)
protocol of Andolfatto et.\ al.\ \citep{Andolfatto2011}.
Following digestion of 50 ng of DNA with the NdeI restriction enzyme, we
ligated each sample to one of 96 adaptors with a unique six base-pair barcode.
After pooling the 96 samples together, we selected 250--300bp fragments to
remain in the library using a Pippen Prep.
For each pool of 96 size-selected samples, we performed eight separate
polymerase chain reactions for 14 cycles (PCR) using Phusion High-Fidelity PCR
Master Mix (NEB Biolabs) and primers that bind to common regions in the
adaptors.
Following PCR, we did two rounds of AMPure XP bead cleanup (Beckman Coulter,
Inc.) using a 0.8 bead volume to sample ratio.
Each library was sequenced in one lane of an Illumina Hiseq 2500 high-output
run, with single-end 100bp reads.
We provide information on all of the individuals included in the three RADseq
libraries in Table~S\ref{table:reftosampledata}, a subset of which were
included in the population pairs we analyzed for this study (Table
\ref{table:comparisons} \& S\ref{table:reftopairsampledata}).
We deposited the demultiplexed, raw sequence reads into the NCBI Sequence Read
Archive
(Bioproject PRJNA486413, SRA Study SRP158258),
and the assembled data matrices are available in our project repository
(\url{https://github.com/phyletica/gekgo}).


\subsection{Data assembly}
We used ipyrad version 0.7.13 \citep{ipyrad0713} to demultiplex and assemble
the raw RADseq reads into loci.
To maximize the number of assembled loci, we de novo assembled the reads
separately for each pair of populations.
All of the scripts and ipyrad parameters files we used to assemble the data are
available in our project repository (\url{https://github.com/phyletica/gekgo}),
and the ipyrad settings are listed in
Table~S\ref{table:ipyradsettings}.

\subsection{Testing for shared divergences}
We approach the inference of temporally clustered divergences as a problem of
model choice.
Our goal is to treat the number of divergence events shared (or not) among the
pairs of populations, and the assignment of the pairs to those events, as
random variables to be estimated from the aligned sequence data.
For eight pairs, there are 4,140 possible divergence models.
I.e., there are 4,140 ways to partition the eight pairs to $\nevents = 1, 2,
\ldots, 8$ divergence events \citep{Bell1934,Oaks2014dpp,Oaks2018ecoevolity}.
Whereas divergences caused by sea-level rise would not happen simultaneously,
we expect that on a timescale of the lizards' mutation rate, treating them as
simultaneous should be a better explanation of data generated by such a
process than treating them as independent.

Given the large number of models, and our goal of making probability statements
about them, we used a Bayesian model-averaging approach.
Specifically, we used the full-likelihood Bayesian comparative biogeography
method implemented in the software package \ecoevolity version 0.1.0 (commit
b9f34c8) \citep{Oaks2018ecoevolity}.
This method models each pair of populations as a two-tipped ``species'' tree,
with an unknown, constant population size along each of the three branches, and
an unknown time of divergence.
This method can directly estimate the likelihood of values of these unknown
parameters from orthologous biallalic characters by analytically integrating
over all possible gene trees and mutational histories \citep{Bryant2012,
    Oaks2018ecoevolity}.
Within this full-likelihood framework, this method uses a Dirichlet process
prior on the assignment of our pairs to an unknown number of divergence times.
The Dirichlet process is specified by a
(1) concentration parameter, \concentration, which determines how probable it
is for pairs to share the same divergence event, \emph{a priori}, and
(2) base distribution, which serves as the prior on the unique divergence
times.

Importantly, because the pairs of populations are modeled as disconnected
species trees, the relative rates of mutation among the pairs is not
identifiable.
This requires us to make informative prior assumptions about the relative rates
of mutation among the pairs.
Because \spp{Cyrtodactylus} and \spp{Gekko} are deeply divergent
\citep[$>$ 80 mya;][]{Gamble2011},
and nothing is known about their relative rates of mutation, we analyzed the
two genera separately.
Within each genus, the populations are all closely related \citep{Welton2010,
    Welton2010zootaxa, Siler2010, Siler2012, Siler2014kikuchii} allowing us to
make the simplifying assumption that the rates of mutation are equal across
pairs \emph{within} each genus.

Based on previous data \citep{Welton2010, Welton2010zootaxa, Siler2010} we
assumed a prior of \dexponential{0.005} on divergence times for our eight pairs
of \spp{Cyrtodactylus} populations, in units of substitutions per site.
To explore the sensitivity of our results to this assumption, we also
tried a prior of \dexponential{0.05} on the divergence times.
Based on previous data \citep{Siler2012, Siler2014kikuchii}, we assumed a prior
of \dexponential{0.0005} on divergence times for our eight pairs of \spp{Gekko}
populations, in units of substitutions per site.
To explore the sensitivity of our results to this assumption, we also tried
priors of \dexponential{0.005} and \dexponential{0.05} on the \spp{Gekko}
divergence times.

For the concentration parameter of the Dirichlet process, we assumed
a hyperprior of $\distgamma(1.1, 56.1)$ for both genera.
This places approximately half of the prior probability on the model
with no shared divergences ($\nevents = 8$).
By placing most of the prior probability on the model of independent
divergences, if we find posterior support for shared divergences, we can be
more confident it is being driven by the data,
as opposed to the prior on divergence times penalizing additional
divergence-time parameters
\citep{Jeffreys1939,Lindley1957,Oaks2012,Oaks2014reply}.
To explore the sensitivity of our results to this assumption, we also
tried a hyperprior of
$\distgamma(1.5, 3.13)$
and
$\distgamma(0.5, 1.31)$.
The former corresponds with a prior mean number of divergence events of five,
whereas the latter places 50\% of the prior probability on the single
divergence ($\nevents = 1$) model.

For all analyses of both the \spp{Cyrtodactylus} and \spp{Gekko} data, we
assumed equal mutation rates among the pairs, a prior distribution of
\dgamma{4.0}{0.004} on the effective effective size of the populations scaled
by the mutation rate ($\epopsize{}\murate{}$), and a
prior distribution of \dgamma{100}{1} on the relative effective size of the
ancestral population (relative to the mean size of the two descendant
populations).
The model implemented in \ecoevolity assumes each character is unlinked (i.e.,
evolved along a gene tree that is independent conditional on the population
tree).
However, by analyzing simulated data, \citet{Oaks2018ecoevolity} showed
the method performs better when all linked sites are used than when data are
excluded to avoid violating the assumption of unlinked sites.
Accordingly, we analyzed all of the sites of our RADseq loci.
The model implemented in \ecoevolity is also restricted to characters with two
possible states (biallelic).
Thus, for sites with three or more nucleotides (hereafter referred to as
polyallelic sites), we compared how sensitive our
results were to two different strategies:
(1) Removing polyallelic sites, and
(2) recoding the sites as biallelic by coding each state as either having the
first nucleotide in the alignment or a different nucleotide.
We assumed the biallelic equivalent of a Jukes-Cantor model of character
substitution \citep{JC1969} so that our results are not sensitive to how
nucleotides are coded as binary
\citep{Oaks2018ecoevolity}.

For each analysis, we ran 10 independent MCMC chains for 150,000 generations,
sampling every 100th generation.
We assessed convergence and mixing of the chains by inspecting the potential
scale reduction factor \citet[the square root of Equation 1.1 in][]{Brooks1998}
and effective sample size \citep{Gong2014} of the log likelihood and all
continuous parameters using the \texttt{pyco-sumchains} tool of \pycoevolity.
We also visually inspected the trace of the log likelihood and parameters over
generations with the program Tracer version 1.6 \citep{Tracer16}.


\section{Results}

\subsection{Data collection and MCMC convergence}
Table~\ref{table:comparisons} summarize the number of individuals sampled for
each pair of islands, along with the number of assembled loci, and the number
of total, variable, and polyallelic characters.
The 10 independent MCMC chains of all our \ecoevolity analyses appeared to have
converged almost immediately.
We conservatively removed the first 101 samples, leaving 1400 samples from each
chain (14,000 samples for each analysis).
With the first 101 samples removed, across all our analyses, all ESS values
were greater than 2000, and all PSRF values were less than 1.005.

\subsection{Testing for shared divergences}

\subsubsection{\spp{Cyrtodactlus} population pairs}
For \spp{Cyrtodactylus}, our \ecoevolity results support the model of no shared
divergences, i.e., all eight pairs of populations diverged independently
(Figures \ref{fig:cyrtnevents} \& \ref{fig:cyrtdivtimes}).
This support is consistent across all three priors on the concentration
parameter of the Dirichlet process
(Figures
S\ref{fig:cyrtneventsbyconcentration}
\&
S\ref{fig:cyrtdivtimesbyconcentration}).
The support is also consistent across both priors on divergence times
and whether polyallelic sites are recoded or removed
(Figures
S\ref{fig:cyrtneventsbytimeprior}
\&
S\ref{fig:cyrtdivtimesbytimeprior}).
Estimates of effective population sizes are also very robust to
priors on \concentration and \divtime, and whether polyallelic sites
are recoded or removed
(Figures
S\ref{fig:cyrtpopsizesbyconcentration}
\&
S\ref{fig:cyrtpopsizesbytimeprior}).

\ifembed{
\input{fig-cyrt-nevents.tex}
}{}

\ifembed{
\input{fig-cyrt-div-times.tex}
}{}

\subsubsection{\spp{Gekko} population pairs}
For \spp{Gekko}, posterior probabilities weakly support no shared divergences,
but Bayes factors weakly support seven divergence events across the eight pairs
(Figure~\ref{fig:gekkonevents}),
suggesting a possible shared divergence between
\spp{G.\ mindorensis} on the Islands of Panay and Masbate
and
\spp{G.\ porosus} on the Islands of Sabtang and Batan
(Figure~\ref{fig:gekkodivtimes}).
Under the intermediate prior on the concentration parameter, support
increases for this shared divergence increases 
(Figures
S\ref{fig:gekkoneventsbyconcentration}
\&
S\ref{fig:gekkodivtimesbyconcentration}).
Under the prior that puts most of the probability on one shared event,
posterior probabilities prefer six divergences
(Figure~S\ref{fig:gekkoneventsbyconcentration})
with another shared divergence between
\spp{G.\ mindorensis} on the Islands of Babuyan Claro and Calayan
and
\spp{G.\ porosus} on the Islands of Romblon and Tablas
(Figure~S\ref{fig:gekkodivtimesbyconcentration}), however,
Bayes factors still prefer seven divergences.
Similarly, as the prior on divergence times becomes more diffuse,
the results shift from ambiguity between seven or eight divergence
events, to ambiguity between six or seven events, to strong
support for six events, with the same island pairs sharing
divergences
(Figures
S\ref{fig:gekkoneventsbytimeprior}
\&
S\ref{fig:gekkodivtimesbytimeprior}).

\ifembed{
\input{fig-gekko-nevents.tex}
}{}

\ifembed{
\input{fig-gekko-div-times.tex}
}{}

As with \spp{Cyrtodactylus}, the estimates of divergence times
are robust to whether polyallelic sites are recoded or removed
(Figure~S\ref{fig:gekkodivtimesbytimeprior}),
and population size estimates are robust to 
priors on \concentration and \divtime, and whether polyallelic sites
are recoded or removed
(Figures
S\ref{fig:gekkopopsizesbyconcentration}
\&
S\ref{fig:gekkopopsizesbytimeprior}).

\section{Discussion}

\subsection{Is there evidence for shared divergences among \spp{Gekko} pairs?}

Under some of the priors we explored, there is support for two possible shared
divergences among the pairs of \spp{Gekko} populations:
(1)
\spp{G.\ mindorensis} on the Islands of Panay and Masbate
and
\spp{G.\ porosus} on the Islands of Sabtang and Batan,
and (2)
\spp{G.\ mindorensis} on the Islands of Babuyan Claro and Calayan
and
\spp{G.\ porosus} on the Islands of Romblon and Tablas.
The Islands of Babuyan Claro and Calayan were never connected,
and we only inferred support for the second shared divergence
under the most extreme priors on \concentration and
\divtime that are expected to favor shared divergences
(Figures
S\ref{fig:gekkoneventsbyconcentration},
S\ref{fig:gekkodivtimesbyconcentration},
S\ref{fig:gekkoneventsbytimeprior},
\&
S\ref{fig:gekkodivtimesbytimeprior}).
Thus, support for the second shared divergence scenario is likely an
artifact of prior sensitivity.
However, the weak support for a shared divergence between
\spp{G.\ mindorensis} on the Islands of Panay and Masbate
and
\spp{G.\ porosus} on the Islands of Sabtang and Batan
under more reasonable priors is interesting because both pairs of islands were
either connected or potentially close enough during glacial periods to allow
gene flow.

Under the priors we initially chose as appropriate (as opposed to those used to
assess prior sensitivity), the posterior probability that the Panay-Masbate and
Sabtang-Batan pairs co-diverged is 0.385.
To evaluate support for this co-divergence, we could calculate a Bayes factor
using the prior probability that any two pairs share the same divergence time,
which is approximately 1.66 in favor of the co-divergence
(Figure~\ref{fig:gekkonevents}).
However, this would not be appropriate, because we did not identify the
Panay-Masbate and Sabtang-Batan pairs of interest \emph{a priori},
but rather our attention was drawn to these pairs based on the posterior
results.
Thus, the probability that \emph{any} two pairs share the same divergence
is no longer the appropriate prior probability for our Bayes factor calculation.
Rather, we need to consider the prior probability that the two pairs with
the most similar divergence times share the same divergence.
To get this prior probability, we can take advantage of the fact that this
condition is met anytime the number of divergence events is less than eight.
Thus, the prior probability that the two pairs with the most similar divergence
times share the same divergence is equal to one minus the prior probability
that all eight pairs diverge independently.
Under our prior on the concentration parameter of $\distgamma(1.1, 56.1)$,
this prior probability is approximately 0.5.
Thus, our posterior probability for the co-divergence between the Panay-Masbate
and Sabtang-Batan pairs is actually \emph{less} than the prior probability,
resulting in a weak Bayes factor of approximately 1.6 in support \emph{against}
the co-divergence.
Thus, based on probability theory, we should favor the explanation that all
eight pairs of \spp{Gekko} populations diverged independently.

\subsection{Implications for the PAIC diversification model}

For each genus we 5--6 pairs of populations that span two islands that were
connected (or nearly so) by terrestrial dry land bridges that formed during
Pleistocene glacial periods.
The connectivity between each pair of islands was likely fragmented by rising
sea levels six or more times over the past three million years
\citep{Rohling1998,Siddall2003}.
Given that, for each genus, we have fewer pairs than the number of times the
islands were fragmented,
the support we found for independent divergence times among the pairs we
analyzed does not obviate all correlates of the PAIC diversification model;
our pairs could have diverged at different fragmentation events.
Comparative genomic data from more pairs of populations would be necessary to
explore this possibility.

Another possibility is that some of our pairs of populations diverged during
the same interglacial period, but the time when gene flow was cut off by rising
sea levels was different enough to be estimated as separate divergences in
\ecoevolity.
Based on estimates of rates of interglacial sea-level rise in Southeast Asia
\citep{Hanebuth2000,Sathiamurthy2006}, it is unlikely that the timing of
island fragmentation across the Philippines would have differed more than 5,000
years.
If we assume a rate of mutation an order of magnitude faster than that
estimated by \citet{Siler2012} for the phosducin gene of Philippine \spp{Gekko}
(1.18\e{-9} substitutions per site per year),
we would not expect to see a difference in divergence times greater than
5.9\e{-6}
between two pairs that diverged during the same interglacial.
It seems reasonable to assume that the difference in divergence times between
\spp{G.\ mindorensis} on the Islands of Panay and Masbate and
\spp{G.\ porosus} on the Islands of Sabtang and Batan
is close to the minimum resolution of \ecoevolity given our data;
there is little posterior variance in the divergence times for these pairs
(Figure~\ref{fig:gekkodivtimes}),
and we note posterior uncertainty in whether these pairs co-diverged or not 
(Figure~\ref{fig:gekkonevents}).
The posterior mean absolute difference in divergence time between these pairs,
conditional on them not co-diverging, is
9.66\e{-6}.
These numbers are similar enough to suggest that it might be possible that
\ecoevolity would have the temporal resolution given our data to distinguish
the divergence times of two pairs that diverged during the same interglacial
fragmentation event.
However, it does not seem very likely, especially when we consider
the strong support inferred for distinct divergences among all but
two of the pairs of populations analyzed. 

\subsection{Sensitivity to the prior on divergence times}

It is interesting that in analyses of both genera we see support for shared
divergences increase as the prior on divergence times becomes more diffuse
(Figures
S\ref{fig:cyrtneventsbytimeprior}
\&
S\ref{fig:gekkoneventsbytimeprior}).
While less extreme here, this is the same pattern seen in
approximate-likelihood Bayesian approaches to this problem
\citep{Oaks2012,Hickerson2013,Oaks2014reply}.
\citet{Hickerson2013} proposed this pattern was caused by numerical problems,
whereas \citet{Oaks2014reply} interpreted the problem as being more
fundamental:
as more prior density is placed in regions of divergence-time space where the
likelihood tends to be low, models that have fewer divergence-time parameters
have greater marginal likelihoods because their likelihoods are ``averaged''
over less space with low likelihood and substantial prior weight.
Our results clearly support the latter explanation, as the MCMC approach used
here does not suffer from the insufficient prior sampling proposed by
\citet{Hickerson2013}.

Whereas the full-likelihood Bayesian approach used here is much more robust to
prior assumptions than the ABC approaches, our results demonstrate that is
still important to assess sensitivity of the results to the priors
\citep{Oaks2012}.
This is especially true for the posterior probabilities of divergence models or
the number of divergence events.
The marginal likelihoods of the divergence models are what update our prior
probabilities to give use these posterior probabilities, and the marginal
likelihood is averaged over the entire parameter space of a model, and weighted
by the prior.
As a result, they can be sensitive to the priors regardless of the
informativeness the data \citep{Oaks2018marginal}.

\subsection{Conclusions}
Climate-driven fragmentation of the Philippine Islands has been invoked as a
model of pulsed co-speciation throughout the archipelago.
This model predicts that population divergences between fragmented islands
should be temporally clustered around interglacial rises in sea levels.
We analyzed comparative genomic data from 16 pairs of insular gecko populations
within a full-likelihood, Bayesian model-choice framework to test for shared
divergence events.
Our results support independent divergences among the pairs of gecko
populations.
While comparative genomic data from more taxa will allow us to address
additional questions, our results suggest the repeated cycles of climate-driven
island fragmentation has not been an important shared mechanism of speciation
for gekkonid lizards in the Philippines.
