\section{Introduction}

An open question in biogeography is to what extent does the landscape, and
changes to it, predict diversification.


\section{Methods}

\subsection{Sampling}
For two genera of geckos, \spp{Cyrtodactylus} and \spp{Gekko}, we sampled
individuals from pairs of populations that occur on two different islands.
Because the climate-mediated fragmentation of the islands was a relatively
recent phenomenon, we selected pairs of populations that were inferred to be
closely related from previous genetic data.
The goal was to avoid deeply divergent pairs that we know \emph{a priori} their
divergence could not have been affected by Late Pliocene and Pleistocene
glacial cycles.
We also sought to sample pairs that span islands that were connected and
fragmented during glacial cycles, and islands that were never connected.
We included the latter as ``controls;'' because these islands were never
connected, the divergence between the populations inhabiting them must be
dispersal mediated, the timing of which is not expected to be clustered across
pairs.
Applying these criteria, we ended up with eight pairs of populations for
\spp{Cyrtodactylus}, four of which, span islands that were connected during
glacial periods (Table \ref{table:comparisons} \& \highLight{SX}).
For \spp{Gekko} we also sampled eight pairs of populations, one of which span
previously connected islands (Table \ref{table:comparisons} \& \highLight{SX}).
\thought{Is it worth including \spp{Gekko}? With only one ``PAIC'' pair, it
    cannot address the hypothesis of interest.}

\subsection{Genomic library preparation and sequencing}

We extracted DNA from tissue using the guanidine thiocyanate method described
by Esselstyn et.\ al.\ \citeyear{Esselstyn2008}.
We diluted the extracted DNA for each individual to a concentration of 5
ng/$\mu$L based on the initial concentration measured with a Qubit 2.0
Fluorometer.
We generated three restriction-site associated DNA sequence (RADseq) libraries,
each with 96 individuals, using the multiplexed shotgun genotyping (MSG)
protocol of Andolfatto et.\ al.\ \citep{Andolfatto2011}.
Following digestion of 50 ng of DNA with the NdeI restriction enzyme, each
sample was ligated to one of 96 adaptors with a unique six base-pair barcode.
After pooling the 96 samples together, we selected 250--300bp fragments to
remain in the library using a Pippen Prep.
For each pool of 96 size-selected samples, we performed eight separate
polymerase chain reactions for 14 cycles (PCR) using Phusion High-Fidelity PCR
Master Mix (NEB Biolabs) and primers that bind to common regions in the
adaptors.
Following PCR, we did two rounds of AMPure XP bead cleanup (Beckman Coulter,
Inc) using a 0.8 bead volume to sample ratio.
Each library was sequenced in one lane of an Illumina Hiseq 2500 high-output
run, with single-end 100bp reads.
We provide information on all of the individuals included in our three RADseq
libraries in \highLight{Table~SX}.

\subsection{Data assembly}
We used ipyrad version 0.7.13 \citep{ipyrad0713} to assemble the raw RADseq
reads into loci.
To maximize the number of assembled loci, we de novo assembled the reads
separately for each pair of populations.
\highLight{Details on ipyrad settings \ldots}

\subsection{Inferring shared divergences}
We approach the inference of temporally clustered divergences as a problem of
model choice.
Our goal is to treat the number of divergence events shared (or not) among the
pairs of populations, and the assignment of the pairs to those events, as
random variables to be estimated from the aligned sequence data.
For eight pairs, there are 4,140 possible divergence models.
I.e., there are 4,140 ways to partition the eight pairs to $\nevents = 1, 2,
\ldots, 8$ divergence events \citep{Bell1934,Oaks2014dpp,Oaks2018ecoevolity}.

Given the large number of models, and our goal of making probability statements
about them, we used a Bayesian model-averaging approach.
Specifically, we used a Dirichlet process prior on the the assignment of our
pairs to an unknown number of divergence times.
We did this using the full-likelihood Bayesian framework implemented in the
software package \ecoevolity \citep{Oaks2018ecoevolity}.
The Dirichlet process is specified by a
(1) concentration parameter, \concentration, which determines how probable it
is for pairs to share the same divergence event, \emph{a priori}, and
(2) base distribution, which serves as the prior on the divergence-time
parameters.

Based on previous data \citep{Welton2010, Welton2010zootaxa, Siler2010} we
assumed a prior of \dexponential{0.005} on divergence times for our eight pairs
of \spp{Cyrtodactylus} populations, in units of subsitutions per site.
To explore the sensitivity of our results to this assumption, we also
tried a prior of \dexponential{0.05} on the divergence times.
We also assumed a hyperprior of $\distgamma(1.5, 3.13)$ on the concentration
parameter of the Dirichlet process, which corresponds to a prior mean number of
divergence events of five.
To explore the sensitiviy of our results to this assumption, we also
tried a hyperprior of
$\distgamma(1.1, 56.1)$
and
$\distgamma(0.5, 1.31)$,
which place 50\% of the prior probability on the
$\nevents = 8$
and
$\nevents = 1$
models, respectively.

\thought{Need blurb about how \ecoevolity assumes unlinked biallelic characters,
    but that based on simulations we analyzed all the sites and tried both
    recoding and removing polyallelic sites.}

Based on previous data \citep{Siler2012, Siler2014kikuchii}, we assumed a prior
of \dexponential{0.0005} on divergence times for our eight pairs of \spp{Gekko}
populations, in units of substitutions per site.
To explore the sensitivity of our results to this assumption, we also tried
priors of \dexponential{0.005}, \dexponential{0.05} on the \spp{Gekko}
divergence times.
For the concentration parameter of the Dirichlet process, we explored the same
three hyperpriors described above.

For all analyses of both the \spp{Cyrtodactylus} and \spp{Gekko} data, we
assumed equal mutation rates among the pairs, a prior distribution of
\dgamma{4.0}{0.004} on the effective effective size of the populations, and a
prior distribution of \dgamma{100}{1} on the relative effective size of the
ancestral population (relative to the mean size of the two descendant
populations).
For each analysis, we ran 10 independent MCMC chains for 150,000 generations,
sampling every 100th generation.

We assessed convergence and mixing of the chains by inspecting the potential scale
reduction factor (the square root of Equation 1.1 in Brooks and Gelman
\citeyear{Brooks1998}) and effective sample size \citep{Gong2014} of the
likelihood and all parameters using the sumchains tool of \pycoevolity.
We also visually inspected the trace of the log likelihood and parameters over
generations with the program Tracer version 1.6 \citep{Tracer16}.


\section{Results}

Figure~\ref{fig:cyrtdivtimes}

Figure~\ref{fig:gekkodivtimes}

Figure~\ref{fig:cyrtnevents}

Figure~\ref{fig:gekkonevents}

Figure~\ref{fig:cyrtpopsizes}

Figure~\ref{fig:gekkopopsizes}

\section{Discussion}
