	<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
	<html>
	<head>
	<title>NGS QC Toolkit</title>
	<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1">
	
	
	<style>
		BODY {
			margin-top: 0;
			background-repeat: repeat;
			font-family: Arial, Helvetica, sans-serif;
			font-size: 5px;
			margin-bottom: 0;
		}
		.cnt {
			font-family: Verdana, Arial, Helvetica, sans-serif;
			font-size: 12px;
			line-height: 20px;
			padding: 5px 20px 0px;
		}
		Table .exp {
			font-family: Verdana, Arial, Helvetica, sans-serif;
			font-size: 13px;
			line-height: 20px;
		}
		TD .padding {
			padding: 0px 20px;
		}
		.head1 {
			font-size: 18px;
			font-weight: bold;
			padding: 5px 0px;
		}
		.head2 {
			font-size: 14px;
			font-weight: bold;
			padding: 5px 0px;
		}
		.head3 {
			font-size: 12px;
			font-weight: bold;
		}
		A {
			text-decoration: none;
			color: #0000FF;
		}
		.tblBg TABLE TD {
			background-color: #EEEEEE;
		}
		.tblBg2 TABLE TD {
			background-color: #E1E1E1;
		}
	</style>

	
	
	</head>
	
	<body bgcolor="#bbbbbb">
	<table align="center" width="900" cellspacing="0" cellpadding="0" bgcolor="#ffffff" border="0">
	
		<tr>
			<td width="17" rowspan="6">&nbsp;</td>
			<td height="150" bgcolor="#E1E1E1"><center><a href="http://www.nipgr.res.in/ngsqctoolkit.html"><b><font  style="font-size: 50px;">NGS QC T</font><font style="font-size: 40px;">OOLKIT</font></b></center></a>
			</td>
			<td width="17" rowspan="6"></td>
		</tr>
		<tr>
			<td valign="top" class="tblBg">
			<div class="cnt">
				<table class="cnt" width="100%" border="0">
					<tr>
						<td class="head1">Results of quality control (QC) using IlluQC.pl v2.3
 <font style="font-size:10px">(Wed Mar 26, 2014)</font></td>
					</tr>
					<tr>
						<td class="head2">Input files and parameters:</td>
					</tr>
					<tr><td class="tblBg2">
						<table width="100%" border="0" class="cnt">
						<tr>
							<td>Analysis type</td><td>Quality check  of single end data</td>
						</tr>
						<tr>
							<td>Input file directory</td><td><a href="file:///Volumes/wdhd/users/joaks1/projects/gekgo/data/genomes/msg/plate3/demultiplexed-raw/fastq/" target="_blank">/Volumes/wdhd/users/joaks1/projects/gekgo/data/genomes/msg/plate3/demultiplexed-raw/fastq/</a></td>
						</tr>
						<tr>
							<td>Input file 1</td><td>CDS_63_Gekko_mindorensis_Panay_R1.fq</td>
						</tr>
						<tr>
							<td>Input file format</td><td>FASTQ (Sanger variant)</td>
						</tr>
						<tr>
							<td>Primer/Adaptor library</td><td>NA
</td>
						</tr>
						<tr>
							<td>Cut-off read length for HQ</td><td>70%
</td>
						</tr>
						<tr>
							<td>Cut-off quality score</td><td>20
</td>
						</tr>
						<tr>
							<td>Only statistics</td><td>On</td>
						</tr>
						<tr>

							<td>Number of processes</td><td>2
</td>

						</tr>
						</table>
					</td>
					</tr>
					<tr>
						<td class="head2">Output files:</td>
					</tr>
					<tr><td class="tblBg2">
						<table width="100%" border="0" class="cnt">
						<tr>
							<td>Output folder</td><td><a href="file:///Volumes/wdhd/users/joaks1/projects/gekgo/data/genomes/msg/plate3/demultiplexed-raw/fastq/qc-output/" target="_blank">/Volumes/wdhd/users/joaks1/projects/gekgo/data/genomes/msg/plate3/demultiplexed-raw/fastq/qc-output/</a></td>
						</tr>
						<tr>
							<td>QC statistics</td><td><a href="CDS_63_Gekko_mindorensis_Panay_R1.fq_stat" target="_blank">CDS_63_Gekko_mindorensis_Panay_R1.fq_stat</a></td>
						</tr>
						<tr>
							<td>Per base average quality score for file 1</td><td><a href="CDS_63_Gekko_mindorensis_Panay_R1.fq_avgQual.png" target="_blank">CDS_63_Gekko_mindorensis_Panay_R1.fq_avgQual.png</a></td>
						</tr>
						<tr>
							<td>Read count (%) per base for different quality ranges for file 1 (Raw)</td><td><a href="CDS_63_Gekko_mindorensis_Panay_R1.fq_QualRangePerBase.png" target="_blank">CDS_63_Gekko_mindorensis_Panay_R1.fq_QualRangePerBase.png</a></td>
						</tr>
						<tr>
							<td>Base composition for file 1</td><td><a href="CDS_63_Gekko_mindorensis_Panay_R1.fq_baseCompostion.png" target="_blank">CDS_63_Gekko_mindorensis_Panay_R1.fq_baseCompostion.png</a></td>
						</tr>
						<tr>
							<td>GC content distribution for file 1</td><td><a href="CDS_63_Gekko_mindorensis_Panay_R1.fq_gcDistribution.png" target="_blank">CDS_63_Gekko_mindorensis_Panay_R1.fq_gcDistribution.png</a></td>
						</tr>
						<tr>
							<td>Quality distribution for file 1</td><td><a href="CDS_63_Gekko_mindorensis_Panay_R1.fq_qualDistribution.png" target="_blank">CDS_63_Gekko_mindorensis_Panay_R1.fq_qualDistribution.png</a></td>
						</tr>
						<tr>
							<td>Summary of QC</td><td><a href="CDS_63_Gekko_mindorensis_Panay_R1.fq_summary.png" target="_blank">CDS_63_Gekko_mindorensis_Panay_R1.fq_summary.png</a></td>
						</tr>
						</table>
					</td>
					</tr>
					<tr>
						<td class="head2" style="background-color: #ffffff;">&nbsp;</td>
					</tr>
					<tr>
						<td class="head2">Results of QC</td>
					</tr>
<tr><td class="head3">QC statistics</td></tr>
<tr><td class="tblBg2"><table width="100%" border="0" class="cnt">
<tr><td>File name</td><td>CDS_63_Gekko_mindorensis_Panay_R1.fq</td></tr>
<tr><td>Total number of reads</td><td>1544886</td></tr>
<tr><td>Total number of HQ reads</td><td>1535357</td></tr>
<tr><td>Percentage of HQ reads</td><td>99.38%</td></tr>
<tr><td>Total number of bases</td><td>146764170</td></tr>
<tr><td>Total number of bases in HQ reads</td><td>145858915</td></tr>
<tr><td>Total number of HQ bases in HQ reads</td><td>145087161</td></tr>
<tr><td>Percentage of HQ bases in HQ reads</td><td>99.47%</td></tr>
<tr><td>Number of Primer/Adaptor contaminated HQ reads</td><td>NA</td></tr>
<tr><td>Total number of HQ filtered reads</td><td>1535357</td></tr>
<tr><td>Percentage of HQ filtered reads</td><td>99.38%</td></tr>
</table></td></tr>
<tr><td class="head3">Detailed QC statistics</td></tr>
<tr><td class="tblBg2"><table width="100%" border="0" class="cnt">
<tr><td>File name</td><td>CDS_63_Gekko_mindorensis_Panay_R1.fq</td></tr>
<tr><td>Minimum read length</td><td>95</td></tr>
<tr><td>Maximum read length</td><td>95</td></tr>
<tr><td>Average read length</td><td>95.00</td></tr>
<tr><td>Total number of reads</td><td>1544886</td></tr>
<tr><td>Total number of reads with non-ATGC bases</td><td>684</td></tr>
<tr><td>Percentage of reads with non-ATGC bases</td><td>0.04%</td></tr>
<tr><td>Total number of bases</td><td>146764170</td></tr>
<tr><td>Total number of HQ bases</td><td>145553773</td></tr>
<tr><td>Percentage of HQ bases</td><td>99.18%</td></tr>
<tr><td>Total number of non-ATGC bases</td><td>4570</td></tr>
<tr><td>Percentage of non-ATGC bases</td><td>0.00%</td></tr>
</table></td></tr>
					<tr>
						<td class="head3">Summary of QC</td>
					</tr>
					<tr>
						<td>
							<table width="100%" border="0" class="cnt">
								<tr>
									<td><div align="justify">Following pie chart shows the summary of QC depicting percentage of high quality, low quality and contaminated reads.</div></td>
								</tr>
								<tr>
									<td align="center"><img src="CDS_63_Gekko_mindorensis_Panay_R1.fq_summary.png" border="1"><br><b>(A)</b></td>
								</tr>
							</table>
						</td>
					</tr>
					<tr>
						<td class="head3">Per base average quality scores</td>
					</tr>
					<tr>
						<td>
							<table width="100%" border="0" class="cnt">
								<tr>
									<td><div align="justify">Following graph(s) show per base average PHRED quality scores for input file,  CDS_63_Gekko_mindorensis_Panay_R1.fq(A), before QC.</div></td>
								</tr>
								<tr>
									<td align="center"><img src="CDS_63_Gekko_mindorensis_Panay_R1.fq_avgQual.png" border="1"><br><b>(A)</b>
								</tr>
							</table>
						</td>
					</tr>
					<tr>
						<td class="head3">Read count (%) per base for different quality score ranges</td>
					</tr>
					<tr>
						<td>
							<table width="100%" border="0" class="cnt">
								<tr>
									<td><div align="justify">Following graph(s) show per base read count (%) for different quality score ranges for input file,  CDS_63_Gekko_mindorensis_Panay_R1.fq(A), before QC.</div></td>
								</tr>
								<tr>
									<td align="center"><img src="CDS_63_Gekko_mindorensis_Panay_R1.fq_QualRangePerBase.png" border="1">
									<br><b>(A)</b>
								</tr>
							</table>
						</td>
					</tr>
					<tr>
						<td class="head3">GC content distribution</td>
					</tr>
					<tr>
						<td>
							<table width="100%" border="0" class="cnt">
								<tr>
									<td><div align="justify">Following graph(s) show number of reads for distinct average GC content (%) ranges for input file,  CDS_63_Gekko_mindorensis_Panay_R1.fq(A), before QC.</div></td>
								</tr>
								<tr>
									<td align="center"><img src="CDS_63_Gekko_mindorensis_Panay_R1.fq_gcDistribution.png" border="1"><br><b>(A)</b>
								</tr>
							</table>
						</td>
					</tr>
					<tr>
						<td class="head3">Quality distribution</td>
					</tr>
					<tr>
						<td>
							<table width="100%" border="0" class="cnt">
								<tr>
									<td><div align="justify">Following graph(s) show number of reads for different average PHRED quality scores for input file,  CDS_63_Gekko_mindorensis_Panay_R1.fq(A), before QC.</div></td>
								</tr>
								<tr>
									<td align="center"><img src="CDS_63_Gekko_mindorensis_Panay_R1.fq_qualDistribution.png" border="1"><br><b>(A)</b>
								</tr>
							</table>
						</td>
					</tr>
					<tr>
						<td class="head3">Base composition</td>
					</tr>
					<tr>
						<td>
							<table width="100%" border="0" class="cnt">
								<tr>
									<td><div align="justify">Following graph(s) show base composition (count) for input file,  CDS_63_Gekko_mindorensis_Panay_R1.fq(A), before QC with percentage of bases at the bottom.</div></td>
								</tr>
								<tr>
									<td align="center"><img src="CDS_63_Gekko_mindorensis_Panay_R1.fq_baseCompostion.png" border="1"><br><b>(A)</b>
								</tr>
							</table>
						</td>
					</tr>
				</table>
			</div>
			</td>
		</tr>
		<tr><td style="font-size: 11px;" align="center" valign="middle" height="25" background="/Users/joaks1/Environment/NGSQCToolkit_v2.3/QC/lib/imgs/btmLine.png" bgcolor="#EEEEEE">For Questions and Suggestions, contact <a href="mailto:mjain@nipgr.res.in">Mukesh Jain (mjain@nipgr.res.in)</a>; <a href="mailto:ravipatel@nipgr.res.in">Ravi Patel (ravipatel@nipgr.res.in)</a></td></tr>
	
		<!--- <tr><td colspan="3"><img src="/Users/joaks1/Environment/NGSQCToolkit_v2.3/QC/lib/imgs/down.png"></td></tr> --->
	</table>
	</body>
	</html>
